atest/acceptance/keywords/click_element.robot

```diff
@@ -36,6 +36,12 @@ Double Click Element Error
     [Setup]    Go To Page "javascript/click.html"
     Double Click Element    id:öööö
 
+Click Element Action Chain
+    [Documentation]   Clicks an element using the 'action_chain' argument without a modifier
```

Comment:
Remove the documentation saying what the test does, it doesn't bring value and only adds maintenance burden. If you want, test name could say: `Click Element With Action Chains`